src/deploy.js

```diff
@@ -1,15 +1,85 @@
-import { spawn } from './utils';
+import AWS from 'aws-sdk';
+import s3 from 's3-node-client';
+import dotenv from 'dotenv';
+
+const path = require('path');
```

Comment:
Why use `require` and not `import`?

Reply:
`path`is not used anymore in the latest changes.